.github/workflows/live-links.yaml

```diff
@@ -10,15 +10,23 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - name: Checkout master
-        uses: actions/checkout@v2
-
       - name: Install linkchecker
-        run: sudo apt update && sudo apt install linkchecker
+        run: sudo pip install LinkChecker
 
       - name: Run linkchecker
-        run: linkchecker --threads 0 --ignore-url '.*&start=.*' --ignore-url /q_auto --ignore-url /fl_sanitize --ignore-url /w_* --ignore-url /h_* --ignore-url /blog --no-warnings https://ubuntu.com
+        run: |
+          linkchecker --recursion-level=2 \
+          	--check-extern --no-warnings https://ubuntu.com \
+          	--ignore-url https://res.cloudinary.com \
+          	--ignore-url q_auto \
+          	--ignore-url fl_sanitize \
+          	--ignore-url c_fill \
+          	--ignore-url e_sharpen \
+          	--ignore-url w_[0-9]* \
+          	--ignore-url h_[0-9]* \
+          	--ignore-url https://ubuntu.com/blog \
+          	--ignore-url https://ubuntu.com/security
 
       - name: Send message on failure
         if: failure()
-        run: curl -X POST -F "workflow=${GITHUB_WORKFLOW}" -F "repo_name=${GITHUB_REPOSITORY}" -F "action_id=${GITHUB_RUN_ID}" ${{ secrets.BOT_URL }}?room=%23web-team
```

Comment:
Live ubuntu.com links should report to use as we maintain those links

Reply:
Not sure what the room name for the Web & Design channel is.. before we used "%23web-team" while if I look at the slug in the url it says "web--design".

I will run a quick test.

Reply:
"web--design" worked.